assets/src/edit-story/components/library/panes/text/textSets/textSet.js

```diff
@@ -114,7 +126,14 @@ function TextSet({ elements, translateY, translateX, ...rest }, ref) {
       onKeyUp={handleKeyboardPageClick}
       {...rest}
     >
-      <TextSetElements isForDisplay elements={elements} />
+      {renderImages ? (
+        <TextSetImg
+          src={`${cdnURL}images/text-sets/${id}.png`}
+          alt={rest['aria-label']}
```

Comment:
Now the surrounding button has an aria-label _and_ the img has an `alt`, both with the same values. A bit redundant.

I think we can remove the `aria-label` from the button.

![Screenshot 2021-07-29 at 16 14 59](https://user-images.githubusercontent.com/841956/127507866-95aecb70-cc51-4d64-995c-224a2eab8ae5.png)

Thoughts?

Reply:
Oh yes, for sure - the default label for a button will be derived from its contents (using the ["accessible name" algorithm](https://www.w3.org/TR/accname-1.1/#mapping_additional_nd_name)) and that includes using alt attributes from images.

You can see the name derived from the contents right here in the console (and that it's overruled by the identical direct label):

<img width="601" alt="Screen Shot 2021-07-29 at 16 44 36" src="https://user-images.githubusercontent.com/637548/127513271-7f04be3d-662e-4b48-a20d-1f68d33884c0.png">

Reply:
Checked with VoiceOver, there is no difference, so we can either do `alt=""` on `img` or extract the `aria-label` from `rest` and so on. I see no harm in empty alt and that would be one liner 🤔

Reply:
Works for me 👍